crm_location/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "version": "14.0.1.0.0",
     "depends": ["crm", "base_location"],
     "data": ["views/crm_lead_view.xml"],
-    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "author": "Odoo Community Association (OCA)",
```

Comment:
Why removing this ? Please revert

Reply:
was a warning saying Odoo Community Association (OCA) not appear in manifest ... 
revert changes done :+1: